src/store.rs

```diff
@@ -461,6 +462,7 @@ pub fn read<const N: usize>(store: impl Store, location: Location, path: &Path)
 }
 
 /// Writes contents to path in location of store.
+#[inline(never)]
```

Comment:
Why not just let the compiler determine this?

Reply:
The compiler is bad at optimising for low stack use.

Reply:
We have tooling in Tock for this. You can pass ` -Z emit-stack-sizes` and then analyse the output, like this: https://github.com/tock/tock/blob/master/boards/Makefile.common#L374

Reply:
Sweet! I'll have to investigate :)